vms/platformvm/state/state.go

```diff
@@ -1525,11 +1512,14 @@ func (s *state) CommitBatch() (database.Batch, error) {
 }
 
 func (s *state) writeBlocks() error {
-	for blkID, stateBlk := range s.addedBlocks {
-		var (
-			blkID = blkID
-			stBlk = stateBlk
-		)
+	for blkID, blk := range s.addedBlocks {
+		blkID := blkID
+
+		stBlk := stateBlk{
+			Blk:    blk,
+			Bytes:  blk.Bytes(),
+			Status: choices.Accepted,
+		}
 
 		// Note: blocks to be stored are verified, so it's safe to marshal them with GenesisCodec
 		blockBytes, err := blocks.GenesisCodec.Marshal(blocks.Version, &stBlk)
```

Comment:
Interesting...so we marshal with status on it rn?

Reply:
`s.addedBlocks` is guaranteed to only contain accepted blocks

Reply:
that comment was more a comment on what we store in the db

Reply:
This PR doesn't change what is on disk - that'll go into `v1.10.6` to limit the number of DB changes in a single release.